### PR TITLE
[AFD] Adds a game over screen

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Adds a game over screen!

## Why It's Good For The Game
We haven't had any, so this hopefully helps a bit! Game over screens are just something neat

## Images of changes
image

## Testing
I died :(
Saw game over screen :D

## Changelog
:cl:
add: Adds a game over screen!
/:cl:

### CODE REVIEW

File: code/game/verbs/webmap.dm [hunk 1]
```diff
@@ -10,3 +10,12 @@
 		return
 
 	usr << link(SSmapping.map_datum.webmap_url)
+
+/client/verb/open_balance_pr()
+	set name = "balance_pr"
+	set hidden = TRUE
+
+	if(tgui_alert(usr, "Do you want to open a balance PR?", "Webmap", list("Yes", "No")) != "Yes")
```
Comment (AyIong) [inline code/game/verbs/webmap.dm:18]:
```suggestion
	if(tgui_alert(usr, "Do you want to open a balance PR?", "Open Balance PR", list("Yes", "No")) != "Yes")
```

File: _maps/map_files/stations/cerestation.dmm [hunk 2]
```diff
@@ -16337,7 +16338,7 @@
 /obj/machinery/computer/general_air_control{
 	dir = 4;
 	name = "Tank Monitor";
-	autolink_sensors = list("n2_sensor" = "Nitrogen", "o2_sensor" = "Oxygen", "co2_sensor" = "Carbon Dioxide", "tox_sensor" = "Toxins", "n2o_sensor" = "Nitrous Oxide", "waste_sensor" = "Gas Mix Tank")
+	autolink_sensors = list("n2_sensor"="Nitrogen","o2_sensor"="Oxygen","co2_sensor"="Carbon Dioxide","tox_sensor"="Toxins","n2o_sensor"="Nitrous Oxide","waste_sensor"="Gas Mix Tank")
```
Comment (Contrabang) [inline _maps/map_files/stations/cerestation.dmm:16341]: your map changes are fucked, please use the latest version of StrongDMM and undo these changes

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/_onclick/hud/fullscreen.dm [hunk 3]
```diff
@@ -209,7 +209,71 @@
...
+	mouse_opacity = MOUSE_OPACITY_ICON
+	layer = 20.9 // Just under the hud layer
+
+/atom/movable/screen/fullscreen/robusted/respawn/Destroy()
+	. = ..()
+	if(robusted && !QDELETED(robusted))
+		qdel(robusted)
+
+/atom/movable/screen/fullscreen/robusted/respawn/Click(location,control,params)
+	to_chat(usr, "Clicked [name]")
```
Comment (Contrabang) [inline code/_onclick/hud/fullscreen.dm:256]:
```suggestion
```

**Review by Contrabang — CHANGES_REQUESTED** [review]
video/image please

File: code/game/verbs/webmap.dm (hunk 1 above)
Comment (DGamerL) [reply code/game/verbs/webmap.dm:18]: Oops

**Comment by DGamerL** [comment]
> video/image please

Will add it tonight

File: _maps/map_files/stations/cerestation.dmm (hunk 2 above)
Comment (DGamerL) [reply _maps/map_files/stations/cerestation.dmm:16341]: Updating SDMM as we speak

File: code/_onclick/hud/fullscreen.dm [hunk 4]
```diff
@@ -209,7 +209,70 @@
...
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	plane = -1
+	var/atom/movable/screen/fullscreen/robusted/robusted
+	var/atom/movable/screen/fullscreen/robusted/respawn/respawn
+	var/atom/movable/screen/fullscreen/robusted/open_pr/open_pr
+	var/timer
+
+/atom/movable/screen/fullscreen/robusted/Initialize(mapload)
+	. = ..()
+	timer =addtimer(CALLBACK(src, PROC_REF(qdel), src), 20 SECONDS)
```
Comment (DGamerL) [inline code/_onclick/hud/fullscreen.dm:230]:
```suggestion
	timer = addtimer(CALLBACK(src, PROC_REF(qdel), src), 20 SECONDS)
```

File: _maps/map_files/stations/cerestation.dmm (hunk 2 above)
Comment (DGamerL) [reply _maps/map_files/stations/cerestation.dmm:16341]: Reverted changes
